source/es/guides/overview/why-cypress.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ¿Por qué Cypress?
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} Aprenderás
+
+- Qué es Cypress y por qué deberías usarlo
+- Nuestra misión y en que creemos
+- Principales recursos de Cypress
+
+{% endnote %}
+
```

Comment:
Video is missing here. See original English version:

```
<!-- textlint-disable -->
{% video youtube LcGHiFnBh3Y %}
<!-- textlint-enable -->
```

Reply:
Hi @jennifer-shehane! Thanks for the comment.
I just added the video.

Reply:
Is it ok now @jennifer-shehane ?